Readme.md

```diff
@@ -120,18 +112,18 @@ re.exec('/bar/baz')
 //=> ['/bar/baz', 'bar/baz']
 ```
 
-#### Custom Match Parameters
+#### Custom Matching Parameters
 
-All parameters can be provided a custom regexp, which overrides the default (`[^\/]+`).
+All parameters can be provided a custom regexp, which overrides the default match (`[^\/]+`). For example, you can match digits in the path:
 
 ```js
-var re = pathToRegexp('/:foo(\\d+)')
+var re = pathToRegexp('/icon-:foo(\\d+).png')
```

Comment:
This example is a bit different to the one above (no optional unnamed group), but happy to merge it anyway since the change is negligible.

Reply:
@blakeembrey I defer to you on that call, as you understand the nuances of the library better that I do. My main suggestion is to remove duplication and explicitly call out the most important features.